src/main/java/uk/gov/hmcts/sptribs/caseworker/event/page/IssueFinalDecisionSelectRecipients.java

```diff
@@ -22,14 +22,19 @@ public void addTo(PageBuilder pageBuilder) {
         pageBuilder
             .page("issueFinalDecisionSelectRecipients", this::midEvent)
             .pageLabel("Select recipients")
-            .label("LabelRecipients","Who should receive this decision notice?")
+            .label("labelIssueFinalDecisionSelectRecipients", "Who should receive this decision notice?")
             .complex(CaseData::getCicCase)
             .readonly(CicCase::getRepresentativeCIC, ALWAYS_HIDE)
             .done()
-            .complex(CaseData::getCaseIssueFinalDecision)
-            .optional(CaseIssueFinalDecision::getRecipientSubjectCIC, "")
-            .optional(CaseIssueFinalDecision::getRecipientRepresentativeCIC,"cicCaseRepresentativeCICCONTAINS \"RepresentativeCIC\"")
-            .optional(CaseIssueFinalDecision::getRecipientRespondentCIC,"")
+            .complex(CaseData::getCicCase)
+            .readonlyWithLabel(CicCase::getFullName, " ")
+            .optional(CicCase::getNotifyPartySubject, "")
+            .label("issueFinalDecisionSelectRecipientsNotifyPartiesRepresentative", "")
+            .readonlyWithLabel(CicCase::getRepresentativeFullName, " ")
+            .optional(CicCase::getNotifyPartyRepresentative, "cicCaseRepresentativeFullName!=\"\" ")
```

Comment:
As above?

Reply:
Also I was told (I think by Santoshini) that the presence of this option should be dependent on whether the representative has been selected to receive details about the case, rather than whether we have details for a representative.  Could this be confirmed?

Reply:
checking the name of the representative does that